### DISCUSSION TITLE
Labo 2 : il manque une couleur dans constants.js

### DISCUSSION THREAD

**Discussion by dylanramos** [web-classroom/web-classroom.github.io#51] [Q&A : Pratique / Labos] [answered]
Dans le fichiers constants.js il y a 7 types de formes mais il n'y a que 6 couleurs. 
Etant donné que dans renderer.js j'utilise l'index de shapeTypes pour trouver la couleur correspondante, j'obtiens un undefined pour le carré (dernier index).
Est-ce que mon approche est fausse ou manque-t'il vraiment une couleur?
Merci pour votre aide.

**Comment by Ovich** [comment, marked as answer]
@dylanramos 

Hello

Oui c'est normale. Les couleurs désignent les joueurs et non pas des shapes. Enfin, elle désignent la couleur des pièces d'un joueur. 

Un joueurs aura toujours la même couleur de pièces

**Reply by dylanramos** [reply]
Ah d'accord je vois. Merci pour ton aide.
